### DISCUSSION TITLE
Hydro v2 特性征集

### DISCUSSION THREAD

**Discussion by undefined-moe** [hydro-dev/Hydro#14] [Ideas]

**Comment by undefined-moe** [comment]
![image](https://user-images.githubusercontent.com/29992205/97778472-60f91f80-1bb2-11eb-8235-0a0fab353afa.png)

用户详情界面显示提交频率图

**Comment by AsakuraMizu** [comment]
给个template吧

**Comment by undefined-moe** [comment]
啥template

**Comment by AsakuraMizu** [comment]
feature request的template啊（）

**Comment by undefined-moe** [comment]
PRS Welcome

**Comment by undefined-moe** [comment]
评论区支持 emoji reaction
